integrations/aircall-basic/actions/create-user.md

```diff
@@ -0,0 +1,51 @@
+# Create User
+
+## General Information
+
+- **Description:** Creates a user in Aircall.
+- **Version:** 0.0.1
+- **Group:** Others
+- **Scopes:**: _None_
+- **Endpoint Type:** Action
```

Comment:
Not sure we advertise this anymore

Reply:
It was in the example readme I was given. I'll check in with @bastienbeurier about it.

Reply:
I think it's worth keeping. It's something we show in the dashboard as well.